docker-compose.yml

```diff
@@ -0,0 +1,13 @@
+# Simple docker-composer.yml
+# You can change container_name, port or volume location
+
+version: '3.3'
+
+services:
+  uptime-kuma:
+    image: louislam/uptime-kuma:1
+    container_name: uptime-kuma
+    volumes:
+      - ./data:/app/data
```

Comment:
I'd would suggest you change this to the non-deprecated syntax (--mount) which is more explicit:

```
volumes:
  - type: bind
    source: ./data
    target: /app/data
```

Besides it could be wise to add this ./data folder to git (with an empty `.gitempty` inside it) plus an entry in .gitignore:
```
data/*
!data/.gitempty
```

Wouldn't a volume be a better choice here?